src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added `AddProcessor` overload which exposes the factory pattern
```

Comment:
```suggestion
* Added `AddProcessor` overload for `OpenTelemetryLoggerOptions` which exposes the factory pattern
```

Reply:
I just pushed a couple tweaks to the CHANGELOG entries.